### PR TITLE
JFB-7

### PR DESCRIPTION
**Pull request by rektasan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,33 +1,40 @@
-HELP.md
-target/
-!.mvn/wrapper/maven-wrapper.jar
-!**/src/main/**/target/
-!**/src/test/**/target/
+.gradle
```
Comment (AtticSiol) [inline .gitignore:1]: add ignore "target" folder

Comment (rektasan) [reply .gitignore:1]: Added .target to .gitignore file.

Comment (AtticSiol) [reply .gitignore:1]: this folder has to "disappear" from git at all:
image

Comment (rektasan) [reply .gitignore:1]: Made sure it is completely ignored using .gitignore.

Comment (rektasan) [reply .gitignore:1]: That was tricky. I guess I needed to revert and stop tracking the target folder.

**Review by AtticSiol — APPROVED** [review]

**Pull request merged**